zips/zip-0227.rst

```diff
@@ -322,19 +322,31 @@ For the ``IssueBundle``:
 
 **Note:** that the commitment is not included in the ``IssuanceAction`` itself. As explained below, it is computed later by the validators and added to the note commitment tree.
 
-Specification: Global Issuance State
-====================================
+Specification: Reference Notes and Global Issuance State
+========================================================
+
+Reference Notes
+---------------
+
+A reference note for a Custom Asset MUST be included by the issuer as the first Note in the first Action of the Issuance Bundle where that Custom Asset is being issued for the first time.
```

Comment:
Do we need to require first Action as well? I think it is good to have it as the first Note for sure.

Reply:
Removed first Action